Asilo5/Ballroom-Beats-UI
# Aw edit pickers

@Asilo5 @spaceplesiosaur 
#### What's this PR do?
- merged updated master into this feature branch
- add styling to dance type and song pick the dropdowns
- changes color of navbar and size
- changed color of buttons to stand out more and makes font bold
#### Where should the reviewer start?
- look at styling in dance type and song pick
#### How should this be manually tested?
- pull down PR in a test branch and open in expoCLI and ios simulator to see changes
#### What are the relevant tickets?
- closes tickets #1  and #2
